recipes/oommf/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+TCLTKVERSION=8.5
+CONDA_ROOT_PATH=$(conda info --root)
```

Comment:
@takluyver - are we using $CONDA_ROOT_PATH for JOOMMF to find the location of conda, to then find OOMMF, or do you think this can be removed?